R/parse.R

```diff
@@ -97,7 +97,18 @@ get_parse_data <- function(text, include_text = TRUE, ...) {
   pd <- as_tibble(
     utils::getParseData(parsed, includeText = include_text),
     .name_repair = "minimal"
-  ) %>%
+  )
+  if (getRversion() < "4.2") {
+    is_unicode_parsing_error <- grepl("^<U\\+[0-9]+>$", pd$text)
+    if (any(is_unicode_parsing_error)) {
+      rlang::abort(
+        "Can't parse input due to unicode restriction in base R. Please ",
+        "upgrade R to style this input. ",
```

Comment:
Maybe you can specify the exact version of R here that the user needs to upgrade to?

Reply:
I adapted it.